tag/conf/request-params.xml

```diff
@@ -1,6 +1,6 @@
 <lst name="defaults">
   <str name="echoParams">explicit</str>
   <str name="fl">score,_store</str>
-  <str name="qf">tag ngram</str>
+  <str name="qf">exactName^10 tag ngram</str>
```

Comment:
https://chatlogs.metabrainz.org/brainzbot/metabrainz/msg/4751414/

```suggestion
  <str name="qf">tag^10 ngram</str>
```

Reply:
Wait, what'd be the point of the copyField then?

Reply:
None.

If used, `copyField` should be accompanied with a `<field name="…" … />` entry which can be used to change the way to match it. See `ngram` for example:

https://github.com/metabrainz/mbsssss/blob/f4ddcb99c3c93dbc7ce332772762fedc8416e6d3/tag/conf/schema.xml#L12-L15

Reply:
Oh, now I got how that works. Well, tag is being used as text, but for this we probably want to use `lowercase`, so I guess in that sense a copyfield would make sense? (since we will *also* want to allow to match as text).